ansible/roles/biocache-cli/tasks/main.yml

```diff
@@ -51,6 +51,14 @@
   tags:
     - biocache-cli
 
+- name: reset biocache java memory default
```

Comment:
This looks like it would cause every use of this ansible file to use 1GB as its maximum heap size (unless I am reading it wrong). 

You should be setting your preferred memory size in a ``biocache_opts`` variable in your ansible inventory file instead. That option will be used to override the defaults (which are already 1G) in ``ansible/roles/biocache-cli/templates/environmentSetupFile.sh``

Reply:
Sorry, I know this was a bit hacky - happy to revert. It slipped my mind that this would find its way into the pull request.  It was a local fix for #137, and only affects the biocache configuration which is otherwise hardcoded as demanding 16GB RAM in the biocache.zip file, which seems a little high for an initial setup.  Changing the biocache_opts values in  the /ansible/inventories/vagrant/biocache-cli-vagrant file doesn't seem to affect this, by the way.

Reply:
16GB is required to run the ALA production copy. However, it isn't clear why it would default to that unless either you are manually creating the zip file using ``createProductionExec.sh`` or someone deployed a version to a maven repository using 16GB instead of the default 4GB. All versions deployed from Travis should get 4GB as the default value in their ``environmentSetupFile.sh``, but unusual things may happen if someone ran ``mvn deploy`` or ``mvn install`` locally with some unusual settings.

Does ``environmentSetupFile.sh`` exist on your system?

Reply:
I have the environmentSetupFile.sh with the 1GB defaults in the biocache-cli/templates folder.  I didn't manually create the zip file.  It does seem like the biocache script hardcodes the memory and hence ignores any other configuration.  
The final command in the script on my ALA-demo server looks like:

`exec "$JAVACMD" $JAVA_OPTS -Xms1g -Xmx1g $BIOCACHE_OPTS -XX:+UseG1GC -Dfile.encoding=UTF8 -Dlog4j.configuration=log4j.xml -Dactors.corePoolSize=8 -Dactors.maxPoolSize=16 -Dactors.minPoolSize=8 -Djava.util.Arrays.useLegacyMergeSort=true -Dactors.maxPoolSize=8 \
  -classpath "$CLASSPATH" \
  -Dapp.name="biocache" \
  -Dapp.pid="$$" \
  -Dapp.repo="$REPO" \
  -Dapp.home="$BASEDIR" \
  -Dbasedir="$BASEDIR" \
  au.org.ala.biocache.cmd.CMD2 \
  "$@" `

(the -Xms1g -Xmx1g entries were 16g each before I replaced them, but should possible be omitted all together to allow the other memory configuration to take effect)